### PR TITLE
renamePropretyValue accepts propertyName

### PR DESCRIPTION
**Pull request by nojvek**

### CODE REVIEW

File: lib/report/util.js [hunk 1]
```diff
@@ -86,10 +86,19 @@ export function renamePropertyType(type) {
...
-export function renamePropertyValue(value) {
+export function renameCountryCode(value) {
   const country = COUNTRIES[String(value).toUpperCase()];
   if (country) {
     return country;
   }
   return value;
 }
+
+export function renamePropertyValue(value, header) {
```
Comment (evnp) [inline lib/report/util.js]: Let's call `header` `propertyName` instead, it's really only a header in the context of insights table

File: lib/report/util.js [hunk 2]
```diff
@@ -86,10 +86,19 @@ export function renamePropertyType(type) {
   }[type] || capitalize(type);
 }
 
-export function renamePropertyValue(value) {
+export function renameCountryCode(value) {
   const country = COUNTRIES[String(value).toUpperCase()];
   if (country) {
     return country;
   }
   return value;
```
Comment (evnp) [inline lib/report/util.js]: you didn't write the original, but while you're here... `return COUNTRIES[String(value).toUpperCase()] || value;` is equivalent and less superfluous

File: readme.md [hunk 3]
```diff
@@ -15,3 +15,12 @@
...
 - Start a server and build continuously: `npm start`
 - Open [http://localhost:8080/examples/style-guide/index.html](http://localhost:8080/examples/style-guide/index.html)
+
+
+### Release Log
+
+#### 0.50.1
+ - renamePropertyValue param signature change to (value, header) - header is a new optional field. 
+    If header is $event / $country_code / $mp_country_code, then value will be expanded. 
+    Otherwise it will be passed through.
```
Comment (evnp) [inline readme.md]: I like this, @jbwyme you have any thoughts?

File: test/util/report.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  it(`expands pre-defined types`, () => {
+    expect(renamePropertyType(`number`)).to.equal(`Integer`);
+    expect(renamePropertyType(`boolean`)).to.equal(`True/False`);
+  });
+
+  it(`capitalizes user defined properties`, () => {
+    expect(renamePropertyType(`template`)).to.equal(`Template`);
+    expect(renamePropertyType(`id`)).to.equal(`Id`);
+  });    
+});
```
Comment (evnp) [inline test/util/report.js]: Thanks for adding these tests unrelated to your stuff, love to see that

File: test/util/report.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+describe(`renamePropertyValue`, () => {
+  it(`returns same value when no header given`, () => {
+    expect(renamePropertyValue(`NO`)).to.equal(`NO`);
+    expect(renamePropertyValue(`mr`)).to.equal(`mr`);
+  });
+
+  it(`expands country code when $country_code is the header`, () => {
+    expect(renamePropertyValue(`GB`, `$mp_country_code`)).to.equal(`United Kingdom`);
+    expect(renamePropertyValue(`US`, `$country_code`)).to.equal(`United States`);
```
Comment (evnp) [inline test/util/report.js]: Want to make one of these `us` or `gb` to test+describe the implicit uppercasing aspect?

File: lib/report/util.js (hunk 2 above)
Comment (nojvek) [inline lib/report/util.js]: Fixed

File: lib/report/util.js (hunk 1 above)
Comment (nojvek) [inline lib/report/util.js]: Done

File: test/util/report.js (hunk 5 above)
Comment (nojvek) [inline test/util/report.js]: Done.

File: test/util/report.js (hunk 4 above)
Comment (nojvek) [inline test/util/report.js]: I still don't know how to use "love to see that" as part of a sentence? Is just just another version of "noiiice!"?

Comment (evnp) [inline test/util/report.js]: Pretty much. Is that an aussie thing? I just know it from https://www.youtube.com/watch?v=rQnYi3z56RE

**Pull request closed without merging**